Lib/ipaddress.py

```diff
@@ -1371,12 +1371,12 @@ def __init__(self, address):
         if isinstance(address, tuple):
             IPv4Address.__init__(self, address[0])
             if len(address) > 1:
-                self._prefixlen = int(address[1])
+                arg = address[1]
             else:
-                self._prefixlen = self._max_prefixlen
+                arg = self._max_prefixlen
 
             self.network = IPv4Network(address, strict=False)
-            self.netmask = self.network.netmask
+            self.netmask, self._prefixlen = self._make_netmask(arg)
```

Comment:
Is it necessary to calculating again using `self._make_netmask`? Is `self.netmask, self._prefixlen = self.network.netmask, self.network.prefixlen` not enough?